pkg/plugins/client_test.go

```diff
@@ -254,9 +255,12 @@ func TestClientWithRequestTimeout(t *testing.T) {
 
 	switch e := err.(type) {
 	case *url.Error:
-		err = e.Err
+		assert.Assert(t, e.Timeout())
```

Comment:
Not sure why this change is in this PR

Reply:
It was failing on this PR; https://github.com/moby/moby/pull/39100#issuecomment-484310240 not 100% sure if it's due to changes in hcsshim, or just a coincidence; I can open it as a separate PR if you prefer

Reply:
moved that commit to a separate PR https://github.com/moby/moby/pull/39168